lib/jekyll/configuration.rb

```diff
@@ -9,6 +9,7 @@ class Configuration < Hash
       # Where things are
       "source"              => Dir.pwd,
       "destination"         => File.join(Dir.pwd, "_site"),
+      "collections_dir"     => "",
```

Comment:
an empty string here is fine. If `site.config["collections_dir"]` is empty, then `File.join` will do the right things with the paths.